### DISCUSSION TITLE
Add a time based filter option

### DISCUSSION THREAD

**Discussion by Kurumindla-Kranthivardhan** [langfuse/langfuse#1454] [Ideas]
### Describe the feature or potential improvement

Hey the current filter option is based on the Date only, can you also try to improve it further to time.

**Comment by clemra** [comment]
Thank you, this is a fair suggestion, especially if you have a ton of traffic. 

We'll take it to our backlog and ping you here once we push it.

**Comment by marcklingen** [comment]
Thanks again for suggesting this. Forgot to update this thread, we’ve added this across the UI.
